server/enterprise.go

```diff
@@ -134,12 +135,15 @@ func (s *Server) getBranchFromForkOrUpstreamRepo(ctx context.Context, serverPR *
 
 func (s *Server) createEnterpriseTestsPendingStatus(ctx context.Context, pr *model.PullRequest) {
 	enterpriseStatus := &github.RepoStatus{
-		State:       github.String("pending"),
+		State:       github.String(statePending),
 		Context:     github.String(s.Config.EnterpriseGithubStatusContext),
 		Description: github.String("TODO as org member: After reviewing please trigger label \"" + s.Config.EnterpriseTriggerLabel + "\""),
 		TargetURL:   github.String(""),
 	}
-	s.createRepoStatus(ctx, pr, enterpriseStatus)
+	err := s.createRepoStatus(ctx, pr, enterpriseStatus)
+	if err != nil {
```

Comment:
1/5 `createRepoStatus` should not return an error and be just fire and forget. Because the caller doesn't care most of the time if it succeeded.

Reply:
I would argue that sometimes I am interested to report it to mattermost. But only on a select few occasions.